service/src/main/java/org/cbioportal/service/impl/StructuralVariantServiceImpl.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2018 The Hyve B.V.
+ * This code is licensed under the GNU Affero General Public License (AGPL),
+ * version 3, or (at your option) any later version.
+ */
+
+/*
+ * This file is part of cBioPortal.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.cbioportal.service.impl;
+
+import java.util.List;
+
+import org.cbioportal.model.StructuralVariant;
+import org.cbioportal.persistence.StructuralVariantRepository;
+import org.cbioportal.service.StructuralVariantService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class StructuralVariantServiceImpl implements StructuralVariantService {
+
+    @Autowired
+    private StructuralVariantRepository structuralVariantRepository;
+    
+    @Override
+    public List<StructuralVariant> fetchStructuralVariants(List<String> molecularProfileIds, 
```

Comment:
would be good to have another endpoint to fetch structural variants by type, "FUSION" being one of the types. The types can be found in the `EVENT_INFO` column. 

Please start a list of SV types. List can be 1 long for now, with only FUSION. Add a webservice that supports a type parameter and use the given type to query the `EVENT_INFO` column.

Reply:
agreed